.github/workflows/devcontainer.yml

```diff
@@ -7,11 +7,18 @@ on:
 jobs:
   devcontainer-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [arm64, amd64]
     steps:
       - name: Checkout
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2.1.0
+
       - name: Build and run dev container task
         uses: devcontainers/ci@57eaf0c9b518a76872bc429cdceefd65a912309b # v0.3.1900000329
         with:
-          runCmd: yarn build
+          runCmd: yarn test
```

Comment:
no, test is too slow. we waste to much time when running tests again here

Reply:
Won't it make more sense to run the tests only here or maybe consolidate the current build workflow  with the devcontainer one?

Reply:
no, tests are already very slow and we don't want them to get slower. arm64 emulation would be magnitudes slower, so tests would probably need hours to run

Reply:
Yup. You are correct. I've checked the time it took for the devcontainer workflow to run -> 45/46 minutes while the regular one took 19 minutes.
I just wonder if `yarn build` is enough for exposing the ARM related errors. How about running a small subset of the unit tests?